packages/dds/tree/src/simple-tree/api/conciseTree.ts

```diff
@@ -16,13 +16,14 @@ import { getUnhydratedContext } from "../createContext.js";
  * @remarks
  * This is "concise" meaning that explicit type information is omitted.
  * If the schema is compatible with {@link ITreeConfigurationOptions.preventAmbiguity},
- * types will be lossless and compatible with {@link TreeBeta.create} (unless the options are used to customize it).
+ * types will be lossless and compatible with {@link TreeAlpha.create} (unless the options are used to customize it).
```

Comment:
neither of these links works yet, but this is the API thats getting added in https://github.com/microsoft/FluidFramework/pull/22566